core/src/integration-test/java/com/scalar/db/storage/StorageAdminIntegrationTestBase.java

```diff
@@ -24,11 +23,12 @@
 import java.util.Iterator;
 import java.util.Map;
 import java.util.Set;
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
 
-@SuppressFBWarnings("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
One of the reasons why I wanted to upgrade `JUnit` is this `Lifecycle.PER_CLASS`. After introducing it, we don't need to keep the storage/transaction manager (like `DistributedStorage` and `DistributedTransactionManager`) instances as static fields.

Please see the following for the details:
https://wesome.org/junit-5-test-instance-class-lifecycle